### DISCUSSION TITLE
Can't find feed when use Google podcasts to subscribe the xml

### DISCUSSION THREAD

**Discussion by laraws** [mxpv/podsync#734] [General]
I used podsync docker to generate subscribe url. And then used Google podcast to subscribe it, but the Google podcast display the result below.

![image](https://user-images.githubusercontent.com/40230381/127977801-50800c6d-a086-4a4f-8a5b-b8ec99fad293.png)

Why did this happen?

**Comment by jpaulickcz** [comment]
I have similar problem, it seems some podcasts app don't like the XML flavor?  I tried forcing the XML into Castro Podcast App and it says "Error importing this feed". But it does work in Apple Podcasts.

**Comment by LucasJanin** [comment]
I have the same issue on my side.
It look like podsync can't generate a RSS file.
Any solution?

Thanks

**Comment by LucasJanin** [comment]
I found a solution by adding customs attributes and validating my XML by this website:

https://www.castfeedvalidator.com/validate.php

I exposed the server on port 80. It's needed for validation.
After this, I was able to add my XML to Castro!

My config file:

```
[server]
port = 9090
hostname = "http://xxxxxxx"

[storage]
  [storage.local]
  data_dir = "/app/data" 

[tokens]
youtube = "xxxxxxxxxx"

[feeds]
    [feeds.ID1]
    url = "https://www.youtube.com/channel/UCJldRgT_D7Am-ErRHQZ90uw"
    update_period = "1h"
    quality = "high" # "high" or "low"
    format = "audio" # "audio", "video" or "custom"
    filters = { title = "Yann Marguet" }
    opml = true
    clean = { keep_last = 20 }
    private_feed = true
    [feeds.ID1.custom]
    title = "Yann Marguet - Moi, ce que j'en dis..."
    description = "Yann Marguet sur France Inter"
    author = "Yann Marguet"
    cover_art = "https://www.radiofrance.fr/s3/cruiser-production/2023/01/834dd18e-a74c-4a65-afb0-519a5f7b11c1/1400x1400_moi-ce-que-j-en-dis-marguet.jpg"
    cover_art_quality = "high"
    category = "Comedy"
    subcategories = ["Stand-Up"]
    lang = "fr"
    ownerName = "Lucas Janin"
    ownerEmail = "xxx@xxxx.com"
```
